games-board/xmahjongg/xmahjongg-3.7-r2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools desktop
+
+DESCRIPTION="friendly GUI version of xmahjongg"
+HOMEPAGE="http://www.lcdf.org/xmahjongg/"
+SRC_URI="http://www.lcdf.org/xmahjongg/${P}.tar.gz"
```

Comment:
I suggest: http -> https for `HOMEPAGE` and `SRC_URI`

Reply:
I handled it.